src/components/Navbar/index.tsx

```diff
@@ -1,10 +1,115 @@
-import { FC } from 'react';
+import Image from 'next/image';
+import { FC, useEffect, useState } from 'react';
+import logoHmc from '/public/logo-hmc.svg';
+import Link from 'next/link';
+import { Button } from '@/components/ui/button';
+import {
+  DropdownMenu,
+  DropdownMenuContent,
+  DropdownMenuItem,
+  DropdownMenuLabel,
+  DropdownMenuSeparator,
+  DropdownMenuTrigger
+} from '@/components/ui/dropdown-menu';
+import { HamburgerMenuIcon } from '@radix-ui/react-icons';
 
 const Navbar: FC = () => {
+  const [scroll, setScroll] = useState(false);
+  useEffect(() => {
+    window.addEventListener('scroll', () => {
+      setScroll(window.scrollY > 10);
+    });
+  }, []);
+
+  const NavListItem = [
+    {
+      id: 1,
+      title: 'Home'
+    },
+    {
+      id: 2,
+      title: 'About'
+    },
+    {
+      id: 3,
+      title: 'Learning Path'
+    },
+    {
+      id: 4,
+      title: 'Testimonial'
+    },
+    {
+      id: 5,
+      title: 'FAQ'
+    }
+  ];
+
```

Comment:
avoid set constant variable like this inside the component, better put that inside the component or set centralized folder for save global constant variabel

> cause it will re-render when state change on that component. we dont really need to alway re render the constant like that

Reply:
@adzaky yeahhh, thats true

Reply:
maybe you can create a `mocks`  or `datas` folder for a global constant variable, then define the data's type with `interface`. so the code like this:
```
export interface NavListItem {
  id?: number;
  title?: string
}

export const NavListItems: NavListItem[] = [
  // Your code
]
```

Reply:
@akwanmaroso Done kak, how about this?

Reply:
@vickyadrii how about this kak? [881943b](https://github.com/hammer-code/lms-fe/pull/4/commits/881943b5bf6c9fda2e0b9b2a4d91a398e6f243c0)

Reply:
Looks good @adzaky . but it's better to separate the files between `interfaces.ts` and `menu.ts`. Create a folder `types`, and then move the file `interface` into `types` folder.  and then change the name file to `types.ts`. thank youuu :)

Reply:
@vickyadrii done kak [54c8d8c](https://github.com/hammer-code/lms-fe/pull/4/commits/54c8d8cef4c7ba00ab5512921c3539c7f28013fa)

Reply:
approved. sikat @adzaky